recipe/meta.yaml

```diff
@@ -1,40 +1,45 @@
-{% set version = "3.11.4" %}
+{% set version = "3.12.0" %}
 {% set major_minor = '.'.join(version.split(".")[:2]) %}
 
 package:
   name: cmake
   version: {{ version }}
 
 source:
-  url: https://cmake.org/files/v{{ major_minor }}/cmake-{{ version }}.tar.gz
+  url: https://gitlab.kitware.com/cmake/cmake/-/archive/v{{ version }}/cmake-v3.12.0.tar.bz2
```

Comment:
The last bit here should be `cmake-v{{ version }}.tar.bz2`

Reply:
Yes sure!